content/en/blog/2022-10-04-prove-the-authenticity-of-the-oci-artifacts/index.md

```diff
@@ -0,0 +1,219 @@
+---
+author: developer-guy
+date: 2022-10-11 12:30:00+00:00
+title: Prove the Authenticity of the OCI Artifacts
+description: "We'll talk about integration of the cosign tool, which is a tool for signing and verifying the given container images, blobs, etc, that we used to prove the authenticity of the OCI Artifacts we manage through the OCIRepository resources."
+url: /blog/2022/10/prove-the-authenticity-of-the-oci-artifacts
+tags: [oci]
+resources:
+- src: "**.{png}"
+  title: "Image #:counter"
+---
+
+Software supply chain attacks are one of the most critical risks threatening today's software and have begun to collapse like a dark cloud over the software industry. The Flux community is one of those communities that always takes precautions against these threats. While we primarily provided the measures we mentioned for our products, we also provided articles that raised users' awareness of what we were doing to protect against these threats. You can find all Flux's security articles [here](/tags/security/). I strongly recommend you to read these articles. And we did not stop here, we also introduced features in our products that can protect users against these threats, and today we will talk about one of these features.
```

Comment:
```suggestion
Software supply chain attacks are one of the most critical risks threatening today's software and have begun to collapse like a dark cloud over the software industry. For the Flux family of projects we are taking precautions against these threats. Apart from implementing security features and best practices, it is important to us to educate our users. You can find all Flux's security articles [here](/tags/security/). Today we will talk about a new security feature.
```

Simplified the language somewhat and removed duplication. To the best of my knowledge we are also not using the term *product* for our OSS projects.